swift/ios-shoppe-demo/Views/StoreViewController.swift

```diff
@@ -39,7 +39,7 @@ class StoreViewController: UICollectionViewController, UICollectionViewDelegateF
 
         setupNavigationBar()
 
-        fsIdentify(id: user.id.uuidString, userInfo: user.infoDict)
+//        fsIdentify(id: user.id.uuidString, userInfo: user.infoDict)
```

Comment:
If this isn't needed anymore, should we just delete this line?

Reply:
oops, removed for debug because it was splitting sessions, added back